angular/projects/spark-angular/src/lib/components/sprk-footer/sprk-footer.component.ts

```diff
@@ -227,9 +227,7 @@ import {
                 </ul>
               </div>
             </div>
-            <div class="sprk-o-Stack__item">
-              <ng-content select="[appStoreLinksSlot]"></ng-content>
-            </div>
+            <ng-content select="[appStoreLinksSlot]"></ng-content>
```

Comment:
good catch

Reply:
hmm. I am seeing that internally this is inside of a stack so why move the stack item directive outside of the component?

Reply:
The social links only show up if there are social links (via ngIf="socialLinks) -- it's possible the the user might not have any socials and only want to use App Store badges. It felt reasonable that they should be able to.

Reply:
moved this as suggested!

Reply:
one of the reasons I moved it out is because we don't want a random empty div if there aren't any appStoreLinks used